frappe/public/js/frappe/views/workspace/workspace.js

```diff
@@ -473,9 +473,10 @@ frappe.views.Workspace = class Workspace {
 			"es-line-edit"
 		);
 		// need to add option for icons in inner buttons as well
-		this.page.add_inner_button(__("Create Workspace"), () => {
-			this.initialize_new_page();
-		});
+		if (frappe.model.can_create("Workspace"))
+			this.page.add_inner_button(__("Create Workspace"), () => {
```

Comment:
This is intentional. Creation of `private` workspace is allowed without any roles.

Reply:
The reason is that private workspaces don't affect anyone else, so everyone by default should be allowed to create it. 

If you want to tweak this maybe add permission to workspace for `Desk User` ?

Reply:
Then site owners can chose to remove it if they want to.